docs/getting-started/github-apps.rst

```diff
@@ -45,47 +44,117 @@ sync with the linked GitHub repository.
+    as separate entries on Dockstore. Each entry within a ``.dockstore.yml`` file corresponds to a unique entry on Dockstore.
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - name: globalAligner
+        subclass: CWL
+        primaryDescriptorPath: /runGlobalAligner.cwl
+        testParameterFiles:
+            - /test/globalAligner.cwl.json
+      - name: localAligner
+        subclass: CWL
+        primaryDescriptorPath: /runLocalAligner.cwl
+        testParameterFiles:
+            - /test/localAligner.cwl.json
+
+A common pattern seen on Dockstore is GitHub repositories that store many workflows. The above ``.dockstore.yml``
+has two entries for workflows. Notice that each entry uses a different name. Names are required if you want 
+multiple workflows registered on Dockstore from a single GitHub repository. The names must be unique between
+entries of the `workflows` array. For each unique name present, an entry will be created on Dockstore.
 
 Service YML File
 +++++++++++++++++
 TODO
 
+Migrating Existing Workflows to use GitHub Apps
+-------------------------------------------------
+If you have already registered GitHub workflows on Dockstore using our old method which required you to refresh, you are still able to use GitHub Apps!
+The migration process is fairly straightforward. First, make sure to install our GitHub App onto the repository for the workflow that you want to migrate.
+Next, create a ``/.dockstore.yml`` file and push it to a branch on your GitHub repo, making sure that the name field matches the existing workflowname.
+If your workflow did not have a workflowname set, simply exclude the field. It is very important that these names match, or else Dockstore won't know which
+workflow to associate the ``/.dockstore.yml`` with.
+
+The pushing of a branch with a valid ``/.dockstore.yml`` should trigger Dockstore to convert your workflow into a GitHub App workflow! New versions will be added/updated/deleted
+as if the workflow was originally added using GitHub Apps.  Existing branches will persist, and you can even individually refresh them on the versions tab of the workflow.
+You can convert these existing workflows to use GitHub Apps by adding a valid ``/.dockstore.yml`` to the corresponding branches on GitHub.
+
+Error Handling
+----------------------------------
+Since Dockstore relies on GitHub to tell us when changes have been made on GitHub, there are chances that the message gets lost or delayed.
+Typically, Dockstore reacts within seconds of a change being made on GitHub, however service disruptions can delay this to a few minutes.
+If a message were to get lost, unfortunately you will need to push to GitHub again. Currently, there is no way to tell on Dockstore whether
+a GitHub message was delayed or lost. We recommend waiting a few minutes and then trying to push again. This will hopefully be changed in the near future.
```

Comment:
I would say "This will be changed in the future", dropping the "hopefully" and the "near"